src/App.tsx

```diff
@@ -58,6 +60,31 @@ const BackgroundProvider: FC<BackgroundProviderProps> = ({ children }) => {
   return <>{children}</>;
 };
 
+interface ThemeProviderProps {
+  children: JSX.Element;
+}
+
+const ThemeProvider: FC<ThemeProviderProps> = ({ children }) => {
```

Comment:
Given that this component just outputs the children without adding anything, I think it would make more sense as a custom hook (`useTheme` or something)

Reply:
I changed it to keep the lines in App.tsx minimal with a external hook. I currently see the option of either squeesing it into antoher component (like the `BackgroundProvider`) or using the ThemeProvider solution.

Reply:
I didn't consider that this requires the context, that makes sense now